docs/root/configuration/listener_filters/proxy_protocol.rst

```diff
@@ -0,0 +1,13 @@
+.. _config_listener_filters_proxy_protocol:
+
+Proxy Protocol
+==============
+
+This listener filter adds support for
+`HAProxy Proxy Protocol <http://www.haproxy.org/download/1.9/doc/proxy-protocol.txt>`_.
+
+In this mode, the upstream connection is assumed to come from a proxy
+which places the original coordinates (IP, PORT) into a connection-string.
```

Comment:
nit: i would refer to it as a preamble or header rather than connection string

Reply:
resolved